### PR TITLE
ti-pru-cgt-installer

### PR DESCRIPTION
**Pull request by leonheldattoradex**
The guys from BeagleBone maintained something like this for a long time. Instead of downloading the source everytime this is needed as a dependency, let's just have a package instead.

### CODE REVIEW

File: ti-pru-cgt-installer/suite/bookworm/ti_cgt_pru_2.3.3_armlinuxa8hf_busybox_installer.sh
Comment (glneo) [inline ti-pru-cgt-installer/suite/bookworm/ti_cgt_pru_2.3.3_armlinuxa8hf_busybox_installer.sh:1]: Why do you wan this huge binary file in the git repo? Why not use wget like we do currently.

File: ti-pru-cgt-installer/version.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export custom_build=true
+export require_root=false
+
+function run_custom_build() {
+    mkdir -p "${builddir}/${package_name}_${deb_version}"
+    cp -r "${debcontroldir}"/* "${builddir}/${package_name}_${deb_version}"
+    (cd "${builddir}/${package_name}_${deb_version}" && debuild --no-lintian -i -us -uc -b)
```
Comment (glneo) [inline ti-pru-cgt-installer/version.sh:9]: We run debuild in the main script. I'm trying to remove this "run_custom_build", you shouldn't use it if not 100% needed.

**Review by glneo — CHANGES_REQUESTED** [review]
Making a new ti-pru-cgt package is fine, but would be better to have the extracted tools in a repo and fetch that using the normal `export git_repo=` in verions.sh mechanism.

**Comment by sskartheekadivi** [comment]
Ah!! Github closed this because you created the PR to my updates branch. @leonheldattoradex, Please create PRs on master if any.

**Comment by leonheldattoradex** [comment]
@sskartheekadivi oh, thanks for the heads-up!
I'll get back on this as soon as possible.

**Pull request closed without merging**